Modelica/Resources/C-Sources/ModelicaInternal.c

```diff
@@ -671,6 +671,13 @@ int ModelicaInternal_getNumberOfFiles(_In_z_ const char* directory) {
 
 _Ret_z_ const char* ModelicaInternal_fullPathName(_In_z_ const char* name) {
     /* Get full path name of file or directory */
+#undef MODELICA_INTERNAL_HAVE_POSIX_REALPATH
+#if defined(_BSD_SOURCE) || \
+    (defined(_XOPEN_SOURCE) && _XOPEN_SOURCE >= 500) || \
+    (defined(_XOPEN_SOURCE) && defined(_XOPEN_SOURCE_EXTENDED)) || \
+    _POSIX_VERSION >= 200112L
```

Comment:
```suggestion
    (defined(_POSIX_VERSION) && _POSIX_VERSION >= 200112L)
```
?

Reply:
Not needed. If _POSIX_VERSION is undefined it is treated as 0 by the pre-processor.
See https://stackoverflow.com/questions/65745333/c-preprocessor-what-is-the-motivation-behind-treating-undefined-macro-as-0 for the reason behind it.

Reply:
And if someone defines it to a non-numeric value it doesn't help.

Reply:
Thanks. Ok as is.